### PR TITLE
Refactored econews comment

### PR DESCRIPTION
**Pull request by WayUp06**

### CODE REVIEW

File: core/src/main/java/greencity/entity/enums/CommentStatus.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package greencity.entity.enums;
+
+public enum CommentStatus {
+    Original,
```
Comment (antonbakalets) [inline core/src/main/java/greencity/entity/enums/CommentStatus.java:4]: Enum names must be in upper case.

File: core/src/main/java/greencity/service/impl/EcoNewsCommentServiceImpl.java [hunk 2]
```diff
@@ -68,7 +68,8 @@ public AddEcoNewsCommentDtoResponse save(Long econewsId, AddEcoNewsCommentDtoReq
      */
     @Override
     public PageableDto<EcoNewsCommentDto> findAllComments(Pageable pageable, User user, Long ecoNewsId) {
-        Page<EcoNewsComment> pages = ecoNewsCommentRepo.findAllByParentCommentIsNullAndEcoNewsIdOrderByCreatedDateDesc(
+        ecoNewsService.findById(ecoNewsId);
```
Comment (antonbakalets) [inline core/src/main/java/greencity/service/impl/EcoNewsCommentServiceImpl.java:71]: Why does `findById` is called and the result of the call is ignored?

File: core/src/main/java/greencity/mapping/EcoNewsCommentDtoMapper.java [hunk 3]
```diff
@@ -3,24 +3,35 @@
...
-            .build();
+        EcoNewsCommentDto dto = new EcoNewsCommentDto();
+        dto.setId(ecoNewsComment.getId());
+        dto.setModifiedDate(ecoNewsComment.getModifiedDate());
+        if (ecoNewsComment.isDeleted()) {
+            dto.setStatus(CommentStatus.Deleted);
+            return dto;
+        }
+
+        if (ecoNewsComment.getCreatedDate().isEqual(ecoNewsComment.getModifiedDate())) {
```
Comment (antonbakalets) [inline core/src/main/java/greencity/mapping/EcoNewsCommentDtoMapper.java:22]: Do I understand correctly that `CommentStatus.Deleted` will be always overridden by `CommentStatus.Original` or `CommentStatus.Edited`?

File: core/src/main/java/greencity/service/impl/EcoNewsCommentServiceImpl.java (hunk 2 above)
Comment (WayUp06) [reply core/src/main/java/greencity/service/impl/EcoNewsCommentServiceImpl.java:71]: It only checks whether econews with this id exists

**Pull request merged**